package.json

```diff
@@ -22,6 +23,10 @@
     "next": "13.2.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.8.0",
     "typescript": "5.0.3"
+  },
+  "devDependencies": {
+    "@types/google.maps": "^3.53.1"
```

Comment:
Is the type definition lib correct?
I think it should be [`@types/google-map-react`](https://www.npmjs.com/package/@types/google-map-react)

Reply:
Yes, it's correct. 
`@types/google-map-react` is the type for another repo (google-map-react)

Reply:
Oh i see